public/locales/en.json

```diff
@@ -251,7 +251,7 @@
     "label_status_tooltip_success_sources_other": "{{count}} Successful Authentications",
     "label_status_tooltip_unreachable": "{{count}} Unreachable System",
     "label_status_tooltip_unreachable_other": "{{count}} Unreachable Systems",
-    "label_summary": "$t(modal.title_scan-summary)",
+    "label_summary": "Scan summary",
```

Comment:
while this works, there's the matter of the (unnecessary) duplication now.

but that's a whole different conversation, and since @nicolearagao mentioned we also need to cleanup this to remove unused strings, we can postpone this conversation.